lonestar/experimental/sweeps/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 if(("${CMAKE_VERSION}" VERSION_GREATER 3.8.0) OR ("${CMAKE_VERSION}" VERSION_EQUAL 3.8.0))
-    app(sweeps sweeps.cpp EXP_OPT)
```

Comment:
Actually this whole if/else can go. I put that in because I was using C++17 in sweeps/triangular solve/ILU before we decided to require C++17 and a more recent CMake.

Reply:
Side note: we should probably come up with a plan for the experimental code. I'll make an issue.

Reply:
Agreed. Thanks.